src/Course/Validation.hs

```diff
@@ -298,6 +321,9 @@ question_Validation_4 = error "todo"
 valueOr :: Validation a -> a -> a
 --                         ^ a fallback value of type `a`
 --         ^^^^^^^^^^^^ potentially some data of type `a`, but potentially not.
+--
+-- Feedback note: ↑`a` and ↓`a` are shadowed here, one as a type and one as a value. It was a bit harder to read.
+--
```

Comment:
Haskell sync note: this is a practice called variable punning where you match the type argument to the variable name.